Dockerfile

```diff
@@ -17,4 +17,4 @@ RUN pip install --no-cache-dir -r requirements.txt
 
```

Comment:
Oh, it looks like `pip install --no-cache-dir` is helpful when working with docker images since pip shouldn't end up reinstalling packages during the lifetime of a container. Nice find, this is probably a good practice to use with our other Dockerfiles based on python